p2p/discovery/mdns.go

```diff
@@ -5,27 +5,26 @@ import (
 	"errors"
```

Comment:
While we're breaking things, can we put this in an `mdns` package? I assume we'll introduce additional discovery mechanisms at some point (we've talked a lot about things like peer exchange).

I don't feel strongly about this.

Reply:
Done.